### PR TITLE
Base pack

### PR DESCRIPTION
**Pull request by liorblob**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/22548

## Dependencies
https://github.com/demisto/demisto-sdk/pull/243

## Description
Create base pack.

### CODE REVIEW

File: Packs/Base/TestPlaybooks/playbook-TestCommonPython.yml [hunk 1]
```diff
@@ -0,0 +1,563 @@
+id: TestCommonPython
```
Comment (yuvalbenshalom) [inline Packs/Base/TestPlaybooks/playbook-TestCommonPython.yml]: shouldn't this one be a move file? I can see any the addition to the pack but not the removal from the general test playbook dir.

**Comment by liorblob** [comment]
@yuvalbenshalom @yaakovi Please review the change I added in `configure_tests.py`.

File: Tests/scripts/configure_tests.py [hunk 2]
```diff
@@ -42,7 +42,8 @@
 # File names
 ALL_TESTS = ["scripts/script-CommonIntegration.yml", "scripts/script-CommonIntegrationPython.yml",
              "scripts/script-CommonServer.yml", "scripts/script-CommonServerUserPython.yml",
-             "scripts/script-CommonUserServer.yml", "scripts/CommonServerPython/CommonServerPython.yml"]
+             "scripts/script-CommonUserServer.yml", "scripts/CommonServerPython/CommonServerPython.yml",
```
Comment (yaakovi) [inline Tests/scripts/configure_tests.py]: i think `scripts/CommonServerPython/CommonServerPython.yml` is not relevant anymore

File: dev-requirements-py3.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 flake8==3.7.9
 bandit==1.6.2
 demisto-py==2.0.9
-git+https://github.com/demisto/demisto-sdk.git@3_10_long_lines_fix
+git+https://github.com/demisto/demisto-sdk.git@master
```
Comment (bakatzir) [inline dev-requirements-py3.txt]:
```suggestion
demisto-sdk==0.4.2
```

**Pull request merged**